src/raft.rs

```diff
@@ -1415,7 +1385,7 @@ impl<T: Storage> Raft<T> {
                     return Ok(());
                 }
 
-                if self.quorum() > 1 {
+                if self.prs().voter_ids().len() > 1 {
```

Comment:
I think it should be `majority(self.prs().voter_ids().len())`.

Reply:
Ah, yes you're right. Because my change means we don't handle a 2 peer cluster well :) Great catch.